packages/discord.js/src/structures/VoiceChannel.js

```diff
@@ -35,6 +35,20 @@ class VoiceChannel extends BaseGuildVoiceChannel {
       permissions.has(PermissionFlagsBits.Speak, false)
     );
   }
+
+  /**
+   * Send a soundboard sound to a voice channel the user is connected to.
+   * @param {SoundboardSound} sound the sound to send
```

Comment:
This ideally should also support passing in just an object with those properties and not just the full structure for docs sake